modernpython/langPack.py

```diff
@@ -54,30 +99,126 @@ def _set_default(text, render):
 def _set_type(text, render):
     return _get_type_and_default(text, render)[0]
 
+def _lower_case_first_char(str):
+    return str[:1].lower() + str[1:] if str else ""
+
+# attributes should never have the same name as a class the may map to
+# Python won't be happy with that, so let's normalize attribute names
+# and leverage aliasing in pydantic to use the cim attribute capitalization
+# during value setting.
+def _set_normalized_name(text, render):
+    return _lower_case_first_char(render(text))
+
+# called by chevron, text contains the label {{dataType}}, which is evaluated by the renderer (see class template)
+def _set_instances(text, render):
+    instance = None
+    try:
+        instance = eval(render(text))
```

Comment:
```suggestion
        # render(text) returns a python dict. Some fileds might be quoted by '&quot;' instead of '"', making the first evel fail.
        instance = ast.literal_eval(render(text))
```

`ast.literal_eval` will not execute arbitrary code, and will only work on well defined python datastructure. Much safer than `eval`.